chat-plugins/dexsearch.js

```diff
@@ -11,13 +11,156 @@
 
 const RESULTS_MAX_LENGTH = 10;
 
-exports.commands = {
-	ds: 'dexsearch',
-	dsearch: 'dexsearch',
-	dexsearch: function (target, room, user, connection, cmd, message) {
-		if (!this.canBroadcast()) return;
+if (!process.send) {
+	// This is the parent
+	let child = exports.child = require('child_process').fork('dexsearch.js', {cwd: __dirname});
```

Comment:
This might not be the right place to export your child... then again, I don't really know of a better place. Chat plugins aren't really "supposed" to have child processes. Maybe try to stick it in `CommandParser.dexsearchProcess`?